src/main/java/uk/gov/hmcts/ccd/domain/service/startevent/DefaultStartEventOperation.java

```diff
@@ -76,12 +80,15 @@ public StartEventResult triggerStartForCaseType(final String caseTypeId,
 
         final CaseTypeDefinition caseTypeDefinition = getCaseType(caseTypeId);
 
+        final Map<String, JsonNode> data = Maps.newHashMap();
+
+        CaseDetails newCaseDetails = caseService
+            .createNewCaseDetails(caseTypeId, caseTypeDefinition.getJurisdictionId(), data);
```

Comment:
probably better having the CaseService.createNewCaseDetails setting the defaults? we could pass the whole `caseTypeDefinition`.
The idea is that all the operations use the CaseService class, and it's the service class the only one responsible for setting the defaults, rather than the various operations. This would eliminate potential bugs where we get the case from the service but forget to set the defaults. And would also reduce duplication

Reply:
We can ignore this comment.
Rafal correctly pointed out that since we consider the defaulting a concept only related to events, it's ok to leave to the Operation the responsibility to do the defaulting, rather than the create methods and get case methods of the `CaseService` class. Because we don't want defaulting to apply every time a case is retrieved.